src/Microsoft.Xaml.Behaviors/NameResolver.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Xaml.Behaviors
     /// <summary>
     /// Provides data about which objects were affected when resolving a name change.
     /// </summary>
-    internal sealed class NameResolvedEventArgs : EventArgs
+    public sealed class NameResolvedEventArgs : EventArgs
```

Comment:
I still struggle to understand why you would need to make this public...

Reply:
An example of its use can be found in https://github.com/MartinEgli/XamlBehaviorsWpf/tree/Behaviors.Extensions/src/Behaviors.Extensions . The extension is in a separate project. A class similar to EventTriggerBase should be created that inherits from TriggerBase and the class should behave similarly. Access to internal elements is required to create a class DependencyPropertyChangedEventTriggerBase. 

The class DependencyPropertyChangedEventTriggerBase also needs access to the NameResolvedEventArgs.